src/EPROMStore.hpp

```diff
@@ -8,164 +8,188 @@
+    // indicates the additional fields that have been stored: 0000 0000 0000 0000
+    //                                                        ^^^^ ^^^^ ^^^^ ^^^^
+    //                                                                        |||
+    //                               UTC Offset (39) -------------------------+||
+    //     DEC lower (31-34) and upper (35-38) limits -------------------------+|
+    //     RA (23-26) and DEC (27-30) Parking offsets --------------------------+
+    //
+    /////////////////////////////////
+
+    enum ItemFlag
+    {
+        MAGIC_MARKER_VALUE
+            = 0xCE00,  // Changed to 0xCxxx in V1.8.60 since we changed RA and DEC Steps to be 10x (previous settings ignored)
+        MAGIC_MARKER_MASK
+            = 0xFE00,  // If these bits are set to MAGIC_MARKER_VALUE, something has been written to the EEPROM
+
+        // The marker bits for the first 8 values stored in EEPROM.
+        RA_STEPS_FLAG       = 0x0001,
+        DEC_STEPS_FLAG      = 0x0002,
+        SPEED_FACTOR_FLAG   = 0x0004,
+        BACKLASH_STEPS_FLAG = 0x0008,
+        LATITUDE_FLAG       = 0x0010,
+        LONGITUDE_FLAG      = 0x0020,
+        PITCH_OFFSET_FLAG   = 0x0040,
+        ROLL_OFFSET_FLAG    = 0x0080,
+        EXTENDED_FLAG       = 0x0100,
+        FLAGS_MASK          = 0x01FF  // The bitwise OR of all preceding values
+    };
+
+    enum ExtendedItemFlag
+    {
+        // The marker bits for the extended values
+        PARKING_POS_MARKER_FLAG = 0x0001,
+        DEC_LIMIT_MARKER_FLAG   = 0x0002,
+        UTC_OFFSET_MARKER_FLAG  = 0x0004,
+    };
+
+    // These are the offsets to each item stored in the EEPROM
+    enum ItemAddress
+    {
+        SPEED_FACTOR_LOW_ADDR       = 0,
+        SPEED_FACTOR_HIGH_ADDR      = 3,  // Split as two discontinuous Uint8
+        HA_HOUR_ADDR                = 1,
+        HA_MINUTE_ADDR              = 2,  // Both Uint8
+        FLAGS_ADDR                  = 4,  // Uint8
+        MAGIC_MARKER_ADDR           = 5,  // Uint8
+        MAGIC_MARKER_AND_FLAGS_ADDR = 4,  // Alias for Uint16 access
+        RA_STEPS_DEGREE_ADDR        = 6,
+        _RA_STEPS_DEGREE_ADDR_1     = 7,  // Int16
+        DEC_STEPS_DEGREE_ADDR       = 8,
+        _DEC_STEPS_DEGREE_ADDR_1    = 9,  // Int16
+        BACKLASH_STEPS_ADDR         = 10,
+        _BACKLASH_STEPS_ADDR_1      = 11,  // Int16
+        LATITUDE_ADDR               = 12,
+        _LATITUDE_ADDR_1            = 13,  // Int16
+        LONGITUDE_ADDR              = 14,
+        _LONGITUDE_ADDR_1           = 13,  // Int16
+        LCD_BRIGHTNESS_ADDR         = 16,  // Uint8
+        PITCH_OFFSET_ADDR           = 17,
+        _PITCH_OFFSET_ADDR_1        = 18,  // Uint16
+        ROLL_OFFSET_ADDR            = 19,
+        _ROLL_OFFSET_ADDR_1         = 20,  // Uint16
+        EXTENDED_FLAGS_ADDR         = 21,
+        _EXTENDED_FLAGS_ADDR_1      = 22,  // Uint16
+        RA_PARKING_POS_ADDR         = 23,
+        _RA_PARKING_POS_ADDR_1,
```

Comment:
Probably should add the explicit numbers here, so they format better.

Reply:
Yeah, or rearrange the enum such that they're in the correct order. We could also add some `static_assert()`s to double check that no-one breaks the order unintentionally